scripts/split-e2e-tests.ts

```diff
@@ -60,6 +60,10 @@ const AWS_REGIONS_TO_RUN_TESTS = [
 const FORCE_REGION_MAP = {
   interactions: 'us-west-2',
   containers: 'us-east-1',
+  'rds-pg-userpool-auth-fields': 'ap-northeast-2',
+  'rds-pg-oidc-auth-fields': 'ap-northeast-2',
+  'rds-mysql-userpool-auth-fields': 'ap-northeast-2',
+  'rds-mysql-oidc-auth-fields': 'ap-northeast-2',
```

Comment:
I believe this can go away once AppSync deployment is done in all regions. Can we add a TODO comment inline or backlog item to remove this later.

Reply:
Added https://app.asana.com/0/1206048977824382/1206394910596509/f